scripts/setup-schemas.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+# This script is used to set-up containers that do not support initialization sequences
+
+# Create Cassandra Schema
+docker exec -i cassandra /bin/bash \
```

Comment:
doing this route, I'd prefer if possible to see if we couldn't do `docker-compose exec cassandra ...` instead of a straight `docker exec` as then we'd not have to give these things names and such.